modules/test/playwright/tests/frontend-data-set-web/main/pages/FDSSamplePage.ts

```diff
@@ -81,19 +108,36 @@ export class FDSSamplePage {
 
 		this.toggleInfoPanelButton = page.getByLabel('Toggle Info Panel');
 
-		const itemActionsCell = this.table.itemActionsCells.first();
+		this.visualizationModeSelector = page.getByLabel('Show View Options');
+	}
 
-		this.itemActionButton = itemActionsCell.getByRole('button', {
-			exact: true,
-			name: 'Actions',
+	async changeVisualizationMode(
```

Comment:
It hurt me add this, as it is a third duplication, but one more refactor in PR would be too much :disappointed:  

What we should do here is to have something like `FDSPage`, that is reused here, in DSM, and elsewhere in other projects. `FDSSamplePage` should contain only bits specific to the sample portlet, like tabs navigation.